README.md

```diff
@@ -253,6 +254,14 @@ curl --request POST \
   ]'
 ```
 
+## Azure Blob Storage
+
+This service uses Azure Blob storage to store the raw artefact data. This is configured in [AzureBlobConfiguration](./src/main/java/uk/gov/hmcts/reform/pip/account/management/config/AzureBlobConfiguration.java).
```

Comment:
Looks like copy and paste error. It is used to store media application image file, not raw artefact data

Reply:
Good spot! Updated